cypress/tools/fixture-generator.py

```diff
@@ -0,0 +1,54 @@
+import json
+import random
+import string
+import uuid
+from datetime import datetime, timedelta
+import os
+
+def random_string(length=8):
+    return ''.join(random.choices(string.printable + 
+                                  string.digits + 
+                                  string.punctuation + 
+                                  string.whitespace, k=length))
+
+def random_date(start, end):
+    delta = end - start
+    random_days = random.randint(0, delta.days)
+    return start + timedelta(days=random_days)
+
+start_date = datetime(2020, 1, 1)
+end_date = datetime(2025, 1, 1)
+
+data = []
+
+for _ in range(15):
+  date = random_date(start_date, end_date)
+
+  random_status = random.choice(['DRAFT', 'FINALIZED'])
+
+  template = {
+        "uri": f"https://api-test.ramanchada.ideaconsult.net/template/{random_string(21)}",
```

Comment:
@sergesoroka Let's put the UUID here, just like in the real templates. Something like:

```py
uuid = str(uuid.uuid4())
template = {
        "uri": f"https://api-test.ramanchada.ideaconsult.net/template/{uuid}",
        "uuid": uuid,
```

Reply:
@sergesoroka Please fix this and regenerate `fixtures/json/bk_rcapi_templates_generated.json`.